packages/expo/cli/utils/user/UserSettings.ts

```diff
@@ -0,0 +1,40 @@
+import JsonFile from '@expo/json-file';
+import os from 'os';
+import path from 'path';
+import process from 'process';
+
+// Extracted from https://github.com/sindresorhus/env-paths/blob/main/index.js
+function getConfigDirectory() {
+  // Share data between eas-cli and expo.
+  const name = 'eas-cli';
```

Comment:
Is this still correct here?

Reply:
ya see comment -- sharing means the settings will be the same across tools.